src/ms.tex

```diff
@@ -252,7 +252,26 @@ \subsection{Challenges}
 Long-term / sustained funding for maintaining infrastructure,
 ...
 
+\section{Contributor Roles}
 
+Taken from \url{https://credit.niso.org/}
+Maybe in Alphabetical order
+\begin{itemize}
+    \item Conceptualization:
+    \item Data curation:
+    \item Formal Analysis:
+    \item Funding acquisition:
+    \item Investigation:
+    \item Methodology:
+    \item Project administration:
+    \item Resources:
+    \item Software: Wolfgang E. Kerzendorf (add ORCID), ADD yourself
+    \item Supervision
+    \item Validation
+    \item Visualization: Andrew Fullard
```

Comment:
Who is Andrew Fullard?

Reply:
my apologies this was just copied over from somewhere else - I removed the names.